### ISSUE TITLE
Issue on upload

### ISSUE THREAD

**Issue by chimon2000** [philschatz/octokat.js#60]
When attempting to upload an asset `release.upload("foo.txt", "adsfdas", "text/plain", bits)`, there's an issue when transfroming the url.

`https://uploads.github.com/repos/chimon2000/test-hello-world/releases/1819396/assets{?name,label}`

becomes

`https://uploads.github.com/repos/chimon2000/test-hello-world/releases/1819396/assets?name,label=foo.txt`

I'm running against the latest version of the GitHub API.

**Comment by philschatz** [comment]
Hmm, maybe GitHub changed their API recently for release assets. The original Pull Request to allow uploading releases is at #26 and it looks like the culprit is [src/replacer.coffee#78](https://github.com/philschatz/octokat.js/pull/26/files#diff-37ecd460d7d441b122a2c553debe0c1cR78).

If just the name is *good enough* then you might be able to just change the line to:

```coffee
# HACK: only use the 1st optional argument. Example: `{?name,label}` for uploads
param = "?#{match[2..-2].split(',')[0]}=#{param}"
```

If you get it working, can you submit a Pull Request?

**Comment by JCMais** [comment]
@philschatz modified it to what you said and it's working, tried to fix it without using that "hack", but I really hate coffeescript with passion. :smile:

If I send a PR just with the modified regex are you going to accept? I would love a new version of octokat.js deployed to npm with this fix, since I'm going to use your project to deploy some binaries for my addon.

**Comment by philschatz** [comment]
I believe this was fixed in #73. If not, please feel free to reopen this issue :smile:
